src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.cs

```diff
@@ -317,6 +317,20 @@ private protected override void OnLoaded()
 			}
 		}
 
+		protected override void OnPointerPressed(PointerRoutedEventArgs args)
+		{
+			if (args.Handled)
+			{
+				base.OnPointerPressed(args);
+				return;
+			}
+
+			if (args.GetCurrentPoint(this).Properties.IsLeftButtonPressed)
+			{
+				args.Handled = Focus(FocusState.Pointer);
+			}
```

Comment:
Why do we need this part? I would not think that a ContentControl can be focused? /cc @MartinZikmund

Reply:
Yes, a lot of ContentControls are focusable. I tested with a bunch of ContentControls to make sure. However, it turns out that that's not general to all ContentControls. ScrollViewer for instance is not focusable, but it tries to grab focus (so that whatever is inside ends up with the focus.) Further more, focus handling should be done inside each element's pointer callbacks, not at the ContentControl level. For example, ButtonBase handles it in OnPointerPressed, but SelectorItems and ScrollViewers do it in OnPointerReleased.
 
 Interestingly, when I was testing ScrollViewer focus in WinUI, a ScrollViewer was indeed being focused, But it turns out that's not the ScrollViewer I was testing, but an "invisible" root ScrollViewer that's used by WinUI windows (thanks to @MartinZikmund for the insight.)

Reply:
Got it. Now that we have general access to winui, are you able to find a more definitive answer about it?

Reply:
ScrollViewer is the one that explicitly asks for focus (like the PR currently does).